### PR TITLE
Add pop-up window

### PR DESCRIPTION
**Pull request by Akhtar-Joomun**
Pop-up window added and project cards HTML tags have been created via javascript

### CODE REVIEW

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested , 

Hi @Akhtar-Joomun , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] please add node_modules and .vsvode in .gitignore file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 1]
```diff
@@ -2,498 +2,406 @@
...
-          <li><a href="#"><i class="fab fa-linkedin-in fa-lg"></i></a></li>
-          <li><a href="#"><i class="fab fa-angellist fa-lg"></i></a></li>
-          <li><a href="#"><i class="fab fa-twitter fa-lg"></i></a></li>
-          <li><a href="#"><i class="fab fa-medium-m fa-lg"></i></a></li>
-        </ul>
-      </section>
-    </footer>
-  </div>
-  <script src="./script.js"></script>
+	<div id="overlay"></div>
```
Comment (arslanbisharat) [inline index.html:31]: - [ ] When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- [ ] When the user clicks (or taps) the close (X) button, the popup disappears.
- [ ] The popup is responsive, it fits mobile and desktop sizes properly.
- [  ] The popup follows the design for both desktop and mobile version from the chosen template in Figma (e.g., font, colors, images, tests, margins).

**Review by arslanbisharat — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Akhtar-Joomun) [reply index.html:31]: Thank you for the review. But I'm not sure you've mentioned what is not working. I think you only mentioned what is working.

DESKTOP VERSION
image




MOBILE VERSION
image


Could you elaborate on the problem/s? 
Thank you for responding.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @Akhtar-Joomun  :wave:

**Great work on making the changes requested by the previous reviewer 👏** 

_You've done well in implementing most of the requested changes, but there are still some which aren't addressed yet._

__However, you need to make some required changes before moving on to the next assignment.__
> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it :blush: 

### **Required Changes :recycle:**
Check my comment below

### **Optional Changes** :zap: 
_Every comment with the **OPTIONAL** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

*Feel free to leave any questions or comments in the PR thread if something is not 100% clear.*

*REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).*

_**Please make the required changes and submit another review request when you are done.**_

* * *

Re-reviewed by: [**Taofeek Olalere**](https://github.com/teekaytech)
*Cheers and Happy Coding* :smiley:

**Comment by Akhtar-Joomun** [comment]
Thank you for the review. 
I'm not sure you are telling me what is wrong with my project though. I have used the same pull request on both submissions. The other pull request is for another project I was working on, while waiting for a response from the first reviewer, which I never got. The reason I made a pull request from the other branch was to check linter errors. I don't think we can check linter errors before creating a pull request, can we?

In the previous one, the reviewer was quite vague and I still tried to find out what was wrong and made some amendments, and then I submitted my project. Now, I don't think I can waste my last submission when I'm not sure what you are asking me to do. I cannot even message you because we don't have private chat here. So, just to be clear, are you asking me to delete my other branch? Or to close the other pull request? 

The bottom line is, I have another pull request, from another branch for the form validation project(If you have a look at it, it contains different codes concerning the form and not any amendment on pop-up window), which I haven't submitted yet. And if you check the comments of the previous reviewer, it's from the same pull request. I am unable to understand why things are so complicated.

I think I'll have to ask for support on this one, if I don't get a response from you. All I need to know, is what is expected from me right now? Delete the pull request or the branch? And is that all that is wrong with my project, so far?

Thank you for your cooperation.

**Comment by teekaytech** [comment]
> Thank you for the review.
> I'm not sure you are telling me what is wrong with my project though. I have used the same pull request on both submissions. The other pull request is for another project I was working on, while waiting for a response from the first reviewer, which I never got. The reason I made a pull request from the other branch was to check linter errors. I don't think we can check linter errors before creating a pull request, can we?
> 
> In the previous one, the reviewer was quite vague and I still tried to find out what was wrong and made some amendments, and then I submitted my project. Now, I don't think I can waste my last submission when I'm not sure what you are asking me to do. I cannot even message you because we don't have private chat here. So, just to be clear, are you asking me to delete my other branch? Or to close the other pull request?
> 
> The bottom line is, I have another pull request, from another branch for the form validation project(If you have a look at it, it contains different codes concerning the form and not any amendment on pop-up window), which I haven't submitted yet. And if you check the comments of the previous reviewer, it's from the same pull request. I am unable to understand why things are so complicated.
> 
> I think I'll have to ask for support on this one, if I don't get a response from you. All I need to know, is what is expected from me right now? Delete the pull request or the branch? And is that all that is wrong with my project, so far?
> 
> Thank you for your cooperation.

Hi @Akhtar-Joomun 

I'm really sorry for what you've experienced so far.
I actually dropped a comment after my review summary, but unfortunately, I did not click on `Start Review` before I submitted the summary. I did not mean to mislead you, kindly accept my apology :pray: 

Please look at the images below and attend to the changes underneath:

**_Figma Design_**
Screenshot from 2021-06-30 21-20-29

**_Your Design_**
clone

It is important that your design is similar to the Figma design as much as possible, therefore:

- [ ] Kindly increase the size of your popup window by reducing the spaces to the left, top and right sides. Doing this will allow the content of the popup to be more legible and look better.
- [ ] You should consider increasing the font size and weight of the popup title, so as to match the design style.
- [ ] Lastly, please make sure that the changes are implemented for both desktop and mobile views.

Again, I'm really sorry for omitting my comment after the review summary: this was not intentional as I already composed the comment but I, unknowingly, did not click on the submit button of the comment :blush: 

Cheers,
Taofeek Olalere.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 
Hi @Akhtar-Joomun, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_
